### DISCUSSION TITLE
2011/08/10/%E8%B1%86%E8%B1%86%E5%85%AB%E4%B8%AA%E6%9C%88

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#343] [Announcements]
# 2011/08/10/%E8%B1%86%E8%B1%86%E5%85%AB%E4%B8%AA%E6%9C%88

时间过得真快，一转眼，豆豆8个月了。

https://blog.qizhen.xyz/2011/08/10/%E8%B1%86%E8%B1%86%E5%85%AB%E4%B8%AA%E6%9C%88

**Comment by ruanqizhen** [comment]
之前博客下的留言：

HY
2011年08月11日 15:35 编辑
把你伤痕累累的躯干撤下去吧，

回复
